frontend/src/app/components/ExperimentPreFeedbackDialog.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import { sendToGA } from '../lib/utils';
+
+export default class ExperimentPreFeedbackDialog extends React.Component {
+
+  render() {
+    const { experiment, surveyURL } = this.props;
+
+    const l10nArgs = JSON.stringify({
+      title: experiment.title
+    });
+
+    return (
+      <div className="modal-container">
+        <div className={classnames('modal', 'tour-modal')}>
+          <header className="tour-header-wrapper">
+            <h3 className="tour-header"
+                data-l10n-id="experimentPreFeedbackTitle"
+                data-l10n-args={l10nArgs}>{experiment.title} Feedback</h3>
+                <div className="tour-cancel" onClick={e => this.cancel(e)}/>
+          </header>
+            <div className="tour-content">
+              <div className="tour-image">
+                <img src={experiment.pre_feedback_image} />
+                <div className="fade" />
+              </div>
+              <div className="tour-text" dangerouslySetInnerHTML={{ __html: experiment.pre_feedback_copy }} />
```

Comment:
Assuming we want markup allowed in `pre_feedback_copy`, [this is the way to do it](https://facebook.github.io/react/tips/dangerously-set-inner-html.html) as far as I understand. It looks ugly as a warning. But, I think it's okay because it's from our own JSON & YAML and we don't need to sanitize our own content.